index.bs

```diff
@@ -24,6 +24,68 @@ Assume Explicit For: yes
 Blah blah blah.
 </section>
 
+<h2 id="deployed-mitigations">
+Some browsers have deployed and announced protections against
+navigational-tracking.  This section is a work in progress to detail what
+protections have been shipped and / or are planned. This section should not
+be understood to be comprehensive.
+
+<h3>Safari</h3>
+Safari uses a heuristic-based approach to combat navigational-tracking. Safari uses
+a client-side machine classifier to determine which sites (determined by
+eTLD+1) are participating in navigational-tracking. The main inputs to this
+classifier are "<a href="//webkit.org/blog/7675/intelligent-tracking-prevention/">
+subresource under number of unique domains, sub frame under number of unique
+domains, and number of unique domains redirected to</a>".
```

Comment:
While the above is true, the relevant detection algorithm for navigational tracking is described under "Protection Against First Party Bounce Trackers" and "Protection Against Tracker Collusion" in https://webkit.org/blog/8311/intelligent-tracking-prevention-2-0/, and "Detection of Delayed Bounce Tracking" in https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/.

Reply:
I've updated the text accordingly.  @johnwilander, if you could see if the three-point criteria i added are correct?